core/pom.xml

```diff
@@ -16,6 +16,18 @@
 
     <dependencies>
         <dependency>
+            <groupId>junit</groupId>
```

Comment:
I moved it here, otherwise Maven would add it to every module as `compile` dependency. However, modules should use it from `testcontainers` dependency as a transitive.